jenkins-slave/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:latest
+MAINTAINER http://fedoraproject.org/wiki/Cloud
+
+RUN dnf -y update && dnf clean all
+RUN dnf -y install java-1.8.0-openjdk openssh-server
```

Comment:
Please add a clean all to this.  you have to clean every layer that you run dnf install on.